pfcpiface/bess.go

```diff
@@ -1143,6 +1278,19 @@ func (b *bess) delFAR(ctx context.Context, done chan<- bool, far far) {
 		}
 
 		b.processFAR(ctx, any, upfMsgTypeDel)
+
+		g := &pb.GtpuPathMonitoringCommandAddDeleteArg{
```

Comment:
Should check enableGtpuPathMonitoring flag here ?

Reply:
Done!